lib/net.js

```diff
@@ -1379,6 +1397,14 @@ Server.prototype.listen = function() {
   var options = normalized[0];
   const cb = normalized[1];
 
+  if (this[serverListenHasBeenCalled]) {
+    var er = new Error('listen method has been called more than once.');
+    er.code = 'ELISTENCALLEDTWICE';
```

Comment:
We don't necessarily need to attach a code. If we really want one, `EISCONN`, `EADDRNOTAVAIL`, or maybe `EALREADY` are options.